run.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -eu
 
-: ${JAVA_HOME:=/usr/lib/jvm/jre-22}
+: ${JAVA_HOME:=/usr/lib/jvm/jre-23}
```

Comment:
Would it be better to make the `run` script more reliable? I wrote this locally, can you either add it or come up with something better?
```bash
if [ -z "${JAVA_HOME}" ]; then
	for path in $(find /usr/lib/jvm -mindepth 1 -maxdepth 1 -type d); do
		if [ "$(echo "${path}" | sed -E 's%/usr/lib/jvm/(java|jre)-([0-9]*).*%\2%')" -ge 22 ]; then
			JAVA_HOME="${path}"
			break
		fi
	done
fi
```

Reply:
Not a bad idea, but it's out of scope for this PR IMO.